sql/connect/common/src/main/protobuf/spark/connect/pipelines.proto

```diff
@@ -104,6 +104,9 @@ message PipelineCommand {
         spark.connect.DataType schema_data_type = 4;
         string schema_string = 5;
       }
+
+      // Optional cluster columns for the table.
+      repeated string cluster_cols = 6;
```

Comment:
Can we use the existing name, `clustering_columns`, consistently instead of adding a new variant, `cluster_cols`, @sryza , @cloud-fan , @gengliangwang ?

https://github.com/apache/spark/blob/ada19082cfe7f091fe9d74b6aee6144673a444ef/sql/connect/common/src/main/protobuf/spark/connect/commands.proto#L144-L145

Reply:
In the `proto` file layer, I hope we can be consistent although this can be different in the API layer (Scala/Java/Python/...)